scalding-core/src/main/scala/com/twitter/scalding/reducer_estimation/RuntimeReducerEstimator.scala

```diff
@@ -8,11 +8,40 @@ import org.apache.hadoop.mapred.JobConf
 import org.slf4j.LoggerFactory
 
 object RuntimeReducerEstimator {
-  val RuntimePerReducer = "scalding.reducer.estimator.runtime.per.reducer"
-  val defaultRuntimePerReducer = 10 * 60 * 1000 // 10 mins
 
   /** Get the target bytes/reducer from the JobConf */
-  def getRuntimePerReducer(conf: JobConf): Long = conf.getLong(RuntimePerReducer, defaultRuntimePerReducer)
+  def getRuntimePerReducer(conf: JobConf): Long = {
+    val key = "scalding.reducer.estimator.runtime.per.reducer"
+    val default = 10 * 60 * 1000 // 10 mins
+    conf.getLong(key, default)
+  }
+
+  /**
+   * Whether to use the median or the mean in the runtime estimation process.
+   * Default is median.
+   */
+  def getRuntimeEstimationScheme(conf: JobConf): RuntimeEstimationScheme = {
+    val key = "scalding.reducer.estimator.runtime.estimation.scheme"
+    val default = "median"
+    conf.get(key, default) match {
+      case "mean" => MeanEstimationScheme
+      case "median" => MedianEstimationScheme
+      case _ => MedianEstimationScheme
```

Comment:
Should we just throw here? I feel it's better for users to be precise if they are providing the scheme than us silently picking median.